docs/REST API/rest-api.md

```diff
@@ -6105,7 +6105,7 @@ The summary can optionally be filtered by team ID.
 }
 ```
 
-### Turn on end user authentication for macOS setup
+### Configure setup experience
```

Comment:
The scope of this endpoint changed in 4.48 to encompass more than just end user authentication so the old heading is not accurate. I took a stab at updating it here, but it would be good for someone from product to weigh in and also to confirm that changing this here won't break other documentation links elsewhere.

Reply:
I think we're good. It doesn't look as though `#turn-on-end-user-authentication-for-macos-setup` is referenced in any other docs.